Update machine-learning.asciidoc - pytorch version dependency (#677)
elastic/eland: docs/guide/machine-learning.asciidoc

[[ml-nlp-pytorch]]
=== Natural language processing (NLP) with PyTorch

> IMPORTANT: You need to appropriate version of PyTorch to import an NLP model. Run
> `python -m pip install 'eland[pytorch]'` to install the latest version.

Reviewer: It's never the latest version, so I'm suggesting this tweak:

```suggestion
IMPORTANT: You need to appropriate version of PyTorch to import an NLP model. Run
`python -m pip install 'eland[pytorch]'` to install that version.
```

Also, README.md has the same wording, would you mind updating it too?

Reviewer: Also there's a missing word that I missed earlier, sorry. You need to **install the** appropriate version.

Author: Changed it in both files! thank you
